src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/MissingKindArgumentToRegisterActionRuleTests.cs

```diff
@@ -217,32 +217,26 @@ End Class
             await VerifyVB.VerifyAnalyzerAsync(source, expected);
         }
 
-        private static DiagnosticResult GetCSharpExpectedDiagnostic(int line, int column, MissingKindArgument kind)
-        {
-            return GetExpectedDiagnostic(line, column, kind);
-        }
+        private static DiagnosticResult GetCSharpExpectedDiagnostic(int line, int column, MissingKindArgument kind) =>
+            VerifyCS.Diagnostic(GetRule(kind))
+                .WithLocation(line, column);
 
-        private static DiagnosticResult GetBasicExpectedDiagnostic(int line, int column, MissingKindArgument kind)
-        {
-            return GetExpectedDiagnostic(line, column, kind);
-        }
+        private static DiagnosticResult GetBasicExpectedDiagnostic(int line, int column, MissingKindArgument kind) =>
+            VerifyVB.Diagnostic(GetRule(kind))
+                .WithLocation(line, column);
 
-        private static DiagnosticResult GetExpectedDiagnostic(int line, int column, MissingKindArgument kind)
+        private static DiagnosticDescriptor GetRule(MissingKindArgument kind)
         {
-            var message = kind switch
+            return kind switch
             {
-                MissingKindArgument.SymbolKind => CodeAnalysisDiagnosticsResources.MissingSymbolKindArgumentToRegisterActionMessage,
+                MissingKindArgument.SymbolKind => CSharpRegisterActionAnalyzer.MissingSymbolKindArgumentRule,
```

Comment:
This needs to use C# and VB specific analyzer rule based on language. I believe I fixed it correctly in my prior PR, so hopefully pulling latest sources should have this fix.